electrum/simple_config.py

```diff
@@ -659,14 +659,15 @@ def get_netaddress(self, key: str) -> Optional[NetAddress]:
             except:
                 pass
 
-    def format_amount(self, x, is_diff=False, whitespaces=False):
+    def format_amount(self, x, is_diff = True, whitespaces = False, add_thousands_sep = True):
```

Comment:
Why change the default value of `is_diff`?

Reply:
also, nit, but please remove the whitespaces around the "=" sign in function arguments (unless the argument is type-annotated) - the rule of thumb is to try to make newly added code look similar to existing code;
but this specific recommendation is also part of [PEP-8](https://www.python.org/dev/peps/pep-0008/#other-recommendations)